### ISSUE TITLE
I'm unable to use the GPU for processing

### ISSUE THREAD

**Issue by ToiYeuTien** [TachibanaYoshino/AnimeGANv3#61]
I have installed all the necessary libraries, and CUDA is also set up, but when I try to convert a video to anime, my computer is still using the CPU for processing. The processing speed is very slow, and I want it to utilize my RTX 3060 GPU. What should I do? Can someone please help?

(AnimeGANv3) G:\Open-AI\AnimeGANv3>nvcc -V
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2021 NVIDIA Corporation
Built on Wed_Jun__2_19:25:35_Pacific_Daylight_Time_2021
Cuda compilation tools, release 11.4, V11.4.48
Build cuda_11.4.r11.4/compiler.30033411_0

(AnimeGANv3) G:\Open-AI\AnimeGANv3>conda list
# packages in environment at D:\.Conda\envs\AnimeGANv3:
#
# Name                    Version                   Build  Channel
absl-py                   2.1.0                    pypi_0    pypi
argparse                  1.4.0                    pypi_0    pypi
astor                     0.8.1                    pypi_0    pypi
bzip2                     1.0.8                h2bbff1b_6
ca-certificates           2024.9.24            haa95532_0
certifi                   2024.8.30                pypi_0    pypi
charset-normalizer        3.4.0                    pypi_0    pypi
colorama                  0.4.6                    pypi_0    pypi
coloredlogs               15.0.1                   pypi_0    pypi
coremltools               6.0                      pypi_0    pypi
cycler                    0.11.0                   pypi_0    pypi
flatbuffers               1.12                     pypi_0    pypi
fonttools                 4.38.0                   pypi_0    pypi
gast                      0.2.2                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.62.3                   pypi_0    pypi
h5py                      3.8.0                    pypi_0    pypi
humanfriendly             10.0                     pypi_0    pypi
idna                      3.10                     pypi_0    pypi
imageio                   2.31.2                   pypi_0    pypi
importlib-metadata        6.7.0                    pypi_0    pypi
joblib                    1.1.0                    pypi_0    pypi
keras-applications        1.0.8                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
kiwisolver                1.4.5                    pypi_0    pypi
libffi                    3.4.4                hd77b12b_1
markdown                  3.4.4                    pypi_0    pypi
markupsafe                2.1.5                    pypi_0    pypi
matplotlib                3.5.3                    pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
networkx                  2.6.3                    pypi_0    pypi
numpy                     1.21.6                   pypi_0    pypi
onnx                      1.10.1                   pypi_0    pypi
onnxruntime               1.1.0                    pypi_0    pypi
onnxruntime-gpu           1.14.1                   pypi_0    pypi
opencv-python             4.6.0.66                 pypi_0    pypi
openssl                   1.1.1w               h2bbff1b_0
opt-einsum                3.3.0                    pypi_0    pypi
packaging                 24.0                     pypi_0    pypi
pillow                    8.4.0                    pypi_0    pypi
pip                       22.1.2             pyhd8ed1ab_0    conda-forge
protobuf                  3.20.1                   pypi_0    pypi
pycryptodome              3.16.0                   pypi_0    pypi
pyparsing                 3.1.4                    pypi_0    pypi
pyqt5                     5.15.9                   pypi_0    pypi
pyqt5-qt5                 5.15.2                   pypi_0    pypi
pyqt5-sip                 12.11.0                  pypi_0    pypi
pyreadline                2.1                      pypi_0    pypi
pyreadline3               3.5.4                    pypi_0    pypi
python                    3.7.16               h6244533_0
python-dateutil           2.9.0.post0              pypi_0    pypi
pywavelets                1.3.0                    pypi_0    pypi
requests                  2.31.0                   pypi_0    pypi
scikit-image              0.17.2                   pypi_0    pypi
scipy                     1.7.3                    pypi_0    pypi
setuptools                65.6.3             pyhd8ed1ab_0    conda-forge
six                       1.16.0                   pypi_0    pypi
sqlite                    3.45.3               h2bbff1b_0
sympy                     1.13.3                   pypi_0    pypi
tensorboard               1.15.0                   pypi_0    pypi
tensorflow-estimator      1.15.1                   pypi_0    pypi
tensorflow-gpu            1.15.0                   pypi_0    pypi
termcolor                 2.3.0                    pypi_0    pypi
tf2onnx                   1.10.1                   pypi_0    pypi
tifffile                  2021.11.2                pypi_0    pypi
tk                        8.6.14               h0416ee5_0
torch                     1.13.1                   pypi_0    pypi
tqdm                      4.66.5                   pypi_0    pypi
typing-extensions         4.7.1                    pypi_0    pypi
tzdata                    2024b                h04d1e81_0
urllib3                   2.0.7                    pypi_0    pypi
vc                        14.40                h2eaa2aa_1
vs2015_runtime            14.40.33807          h98bb1dd_1
werkzeug                  2.2.3                    pypi_0    pypi
wheel                     0.37.1             pyhd3eb1b0_0
wrapt                     1.16.0                   pypi_0    pypi
xz                        5.4.6                h8cc25b3_1
zipp                      3.15.0                   pypi_0    pypi
zlib                      1.2.13               h8cc25b3_1
